src/ansys/mapdl/core/launcher.py

```diff
@@ -1544,10 +1545,19 @@ def launch_mapdl(
 def check_mode(mode: ALLOWABLE_MODES, version: ALLOWABLE_VERSION_INT):
     """Check if the MAPDL server mode matches the allowable version
 
-    If ``None``, the newest mode will be selected.
+    If :class:`None`, the newest mode will be selected.
 
     Returns a value from ``ALLOWABLE_MODES``.
     """
+    if not mode and not version:
+        return "grpc"
+    elif not version:
+        warnings.warn(
+            "PyMAPDL couldn't not detect MAPDL version, hence it could not "
```

Comment:
I might be wrong but the message is a bit confusing.
```suggestion
            "PyMAPDL couldn't detect MAPDL version, hence it could not "
```